docs/modules/operation/pages/deep-dive/alarms/configuring-alarms.adoc

```diff
@@ -122,37 +124,38 @@ When configuring a resolution alarm, set this attribute to match the reduction-k
       ...
       <alarm-data reduction-key="%uei%:%dpname%:%nodeid%:%interface%"
                   alarm-type="2"
-                  clear-key="uei.opennms.org/nodes/interfaceDown:%dpname%:%nodeid%:%interface%"<2>
+                  clear-key="uei.opennms.org/nodes/interfaceDown:%dpname%:%nodeid%:%interface%" <2>
                   auto-clean="false"/>
    </event>
 ----
 <1> The `interfaceDown` event sets a `reduction-key` that includes enough information to identify a specific interface on a specific node.
-<2> The `interfaceUp` has a `clear-key` that matches the `reduction-key` of an `interfaceDown` alarm, allowing a match to automatically clear the previous alarm.
+<2> The `interfaceUp` event has a `clear-key` that matches the `reduction-key` of an `interfaceDown` alarm, letting a match automatically clear the previous alarm.
 
-== Auto-clean attribute
-This attribute instructs Alarmd to retain only the most recent event reduced into an alarm.
-For alarms that are super chatty, this is a way to reduce the size of the most recent events in the database.
+=== auto-clean
+
+The `auto-clean` attribute instructs alarmd to retain only the most recent event that has been reduced into an alarm.
+For alarms that produce many events, this serves as a way to reduce the size of the most recent events in the database.
 
 WARNING: Do not use this feature with alarms that have pairwise correlation (matching problems with resolutions).
```

Comment:
```suggestion
WARNING: Avoid using this feature with alarms that have pairwise correlation (matching problems with resolutions).
It may delete all problem events, erasing your ability to study an alarm's history.
```